templates/menu_local_tasks/menu-local-tasks.tpl.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Contains template file.
+ */
+?>
+<?php if ($primary): ?>
+  <nav class="ecl-navigation-wrapper">
+    <h2 class="ecl-sr-only"><?php print t('Primary tabs') ?></h2>
```

Comment:
I'm a bit worried about adding string values directly into the tpl. Although its not a blocker I would like to know the opinion of the rest on this